mizar/common/common.py

```diff
@@ -415,3 +415,10 @@ def conf_list_has_max_elements(conf, conf_list):
     if (counter + item_len > CONSTANTS.MAX_CLI_CHAR_LENGTH):
         return True
     return False
+
+def bindmount_netns(src_netns_path, dst_netns_path):
+    import os
```

Comment:
Shall we move these "import" to the beginning to keep style consistent?

Reply:
will update as suggested.